lake-manifest.json

```diff
@@ -40,5 +40,13 @@
     "opts": {},
     "name": "proofwidgets",
     "inputRev?": "v0.0.21",
+    "inherited": false}},
+  {"git":
+   {"url": "https://github.com/leanprover/std4",
```

Comment:
Std4 also has an entry at the top of the file. But the `inputRev?` field is different. Is this quasi-duplication intentional?

Reply:
No, an accident, now resolved.